charts/templates/karmada_scheduler_estimator.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if and (eq .Values.installMode "component") (has "schedulerEstimator" .Values.components) }}
+{{ $namespace := include "karmada.namespace" .}}
+{{ $clusterName := .Values.schedulerEstimator.clusterName }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: karmada-scheduler-estimator-{{ $clusterName }}
+  namespace: {{ $namespace }}
+  labels:
+    cluster: {{ $clusterName }}
+    {{- include "karmada.schedulerEstimator.labels" . | nindent 4}}
+spec:
+  replicas: {{  .Values.schedulerEstimator.replicaCount }}
+  selector:
+    matchLabels:
+      app: karmada-scheduler-estimator-{{ $clusterName }}
+      {{- include "karmada.schedulerEstimator.labels" . | nindent 6}}
+  template:
+    metadata:
+      {{- with .Values.schedulerEstimator.podAnnotations }}
+      annotations:
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        app: karmada-scheduler-estimator-{{ $clusterName }}
+        {{- include "karmada.schedulerEstimator.labels" . | nindent 8}}
+        {{- include "karmada.schedulerEstimator.podLabels" . | nindent 8}}
+    spec:
+      {{- with .Values.schedulerEstimator.tolerations}}
+      tolerations:
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: karmada-scheduler-estimator
+          image: "{{ .Values.schedulerEstimator.image.repository }}:{{ .Values.schedulerEstimator.image.tag | default "latest" }}"
+          imagePullPolicy: {{ .Values.schedulerEstimator.image.pullPolicy }}
+          command:
+            - /bin/karmada-scheduler-estimator
+            - --kubeconfig=/etc/{{ $clusterName }}-kubeconfig
+            - --cluster-name={{ $clusterName }}
+          volumeMounts:
+            - name: member-kubeconfig
+              subPath: {{ $clusterName }}-kubeconfig
+              mountPath: /etc/{{ $clusterName }}-kubeconfig
+      volumes:
+        - name: member-kubeconfig
+          secret:
+            secretName: {{ $clusterName }}-kubeconfig
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: karmada-scheduler-estimator-{{ $clusterName }}
+  namespace: {{ $namespace }}
+  labels:
+    cluster: {{ $clusterName }}
+spec:
+  selector:
+    app: karmada-scheduler-estimator-{{ $clusterName }}
+    {{- include "karmada.schedulerEstimator.labels" . | nindent 4}}
+  ports:
+    - protocol: TCP
+      port: 10352
+      targetPort: 10352
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $clusterName }}-kubeconfig
+  namespace: {{ $namespace }}
+stringData:
+  kubeconfig: |-
```

Comment:
An error occurs:

```bash
# kubectl logs -n karmada-system karmada-scheduler-estimator-member1-fcf4b9dc7-2qd6v                         
I0928 03:33:42.008286       1 scheduler-estimator.go:43] karmada-scheduler-estimator version: version.Info{GitVersion:"v0.8.0-151-g741f6839", GitCommit:"741f683945eb3c7a3540d7eeb708d7e2f015a290", GitTreeState:"clean", BuildDate:"2021-09-27T12:37:02Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"linux/amd64"}
error building kubeconfig: error loading config file "/etc/member1-kubeconfig": read /etc/member1-kubeconfig: is a directory
```

So `kubeconfig` should be `{{ $clusterName }}-kubeconfig`. The key name is`{{ $clusterName }}-kubeconfig`. Here is the reference https://github.com/karmada-io/karmada/blob/1947cf219979c728f1aa63fd185b33efd821b1de/hack/deploy-scheduler-estimator.sh#L57.